lib/matplotlib/contour.py

```diff
@@ -1180,15 +1180,34 @@ def _autolev(self, N):
         one contour line, but two filled regions, and therefore
         three levels to provide boundaries for both regions.
```

Comment:
Suspect at least this docstring needs more info....

Reply:
Not sure what info you are looking for, but I am adding some now.  This is a private method, so the docstring requirements are not as stringent as they would be for a public method.